.github/workflows/rubocop.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2.3.4
```

Comment:
I think we should use `v2` instead of a specific version.

Reply:
I think having an explicit version is better, this way you cannot be surprised by a bug or change introduced in v2. These happen.

This PR is a nice workflow - we immediately see confirmation that it actually worked (well except Ruby 3.0+ that is another story).

I suggest to merge.

Reply:
I think it's a trade off between noise in the git log and updates. I wouldn't expect the official checkout action to really break. Otherwise a lot of actions break.